identity/backend/src/main/java/io/camunda/identity/config/IdentityPresets.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(value = "camunda.presets.identity")
+public class IdentityPresets {
+  private String user;
+  private String password;
```

Comment:
Having only one user configurable is an unneeded limitation. Let's make it an array of users.